examples/index.js

```diff
@@ -1,11 +1,23 @@
 /* eslint no-console: "off" */
+const socketIoPort = 2222;
 
 const express = require('express');
 
+// This is optional. If your server uses socket.io already, pass it to config as `webserver` along with it's port.
+const socketio = require('socket.io')(socketIoPort);
```

Comment:
it is not possible to get the port that socket.io uses internally from the socket.io instance? I am trying to find out if that's possible 😉